source/atlas-cli-tutorials.txt

```diff
@@ -0,0 +1,46 @@
+.. _atlas-cli-tutorials:
+
+=======================================
+Manage |service| from the {+atlas-cli+}
+=======================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Use the following tutorials to learn how to manage |service| using the 
+{+atlas-cli+}. 
+
+To learn the {+atlas-cli+} basics, see :ref:`install-atlas-cli` and
+:ref:`connect-atlas-cli`.
+
+You can also go straight to the :doc:`{+atlas-cli+} Commands 
+</command/atlas>`.
```

Comment:
[optional/ suggestion] I think it might read more smoothly to put the table right after "Use the following tutorials..." and then end the page with "To learn the basics" and "You can also..." 

Totally up to you, it's clear as-is